File: 2076-SumOfDigitsOfStringAfterConvert/2076-SumOfDigitsOfStringAfterConvert.py
# Last updated: 6/22/2026, 12:51:37 PM
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        alp = '_abcdefghijklmnopqrstuvwxyz'
        s = ''.join([str(alp.index(i)) for i in s])
        for i in range(k):
            s = str(sum([int(i) for i in s]))
        return int(s)

File: 2076-SumOfDigitsOfStringAfterConvert/2076-SumOfDigitsOfStringAfterConvert.js
// Last updated: 6/22/2026, 12:51:42 PM
/**
 * @param {string} s
 * @param {number} k
 * @return {number}
 */
var getLucky = function(s, k) {
    let str = '_abcdefghijklmnopqrstuvwxyz'
    s = s.split('').reduce((a,b)=>a+str.indexOf(b).toString(),'')  
    for(let i=0; i<k; i++){
        s = s.split('').reduce((a,b)=>parseInt(a)+parseInt(b)).toString()
    }
    return s
};